app/mailers/audit_mailer.rb

```diff
@@ -10,7 +10,7 @@ def summary(options = {})
     @audits  = Audit.authorized_as(user, :view_audit_logs, Audit).search_for(@query).limit(@limit)
 
     set_locale_for(user) do
-      mail(:to => user.mail, :subject => _("Audit summary")) do |format|
+      mail(:to => utf8_encode_email(user.mail), :subject => _("Audit summary")) do |format|
```

Comment:
Can you move the encoding/decoding into ApplicationMailer? Other plugins won't know they have to change this, it can be rewritten automatically in `mail`.